easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'PythonBundle'
+name = 'python-irodsclient'
+version = '1.1.4'
+homepage = 'https://github.com/irods/python-irodsclient'
+description = "A python API for iRODS"
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+]
+
+use_pip = True
+# order is important!
+exts_list = [
+    ('defusedxml', '0.7.1', {}),
+    ('prettytable', '3.3.0', {}),
+    ('python-irodsclient', '1.1.4', {'modulename':'irods'})

```

Comment:
@jefscheepers Thanks a lot for the contribution!

You will need to add SHA256 checksums for each of the extensions here to make the CI tests pass (`eb --inject-checksums` is your friend here), and we would also appreciate from trivial style changes to make the easyconfig match the usual style in easyconfigs a bit better.

A good example to compare with is https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/GitPython/GitPython-3.1.24-GCCcore-11.2.0.eb

Do let us know if you need any help with this!